docs/storage-management.asciidoc

```diff
@@ -107,8 +99,8 @@ INFO      "delete_indices" action completed.
 [float]
 ==== Delete data matching a query
 
-In case you want to delete documents matching a specific query, e.g. all documents with a given `context.service.name`,
-you can do this by sending the following request:
+To delete documents matching a specific query, e.g. all documents with a given `context.service.name`,
```

Comment:
Suggested edit:

To delete documents matching a specific query, for example all documents with a given `context.service.name`, use the following request: